src/graph.rs

```diff
@@ -3767,9 +3868,79 @@ impl<'a, 'graph> Builder<'a, 'graph> {
     }
     module_slot
   }
+
+  #[cfg(feature = "fast_check")]
+  fn create_fast_check_type_graph(&mut self) {
```

Comment:
This currently doesn't support caching. I will add that in https://github.com/denoland/deno_graph/issues/347 as this PR is already large enough.